### DISCUSSION TITLE
Request for Permanent Contribution Visibility after Losing Access to Private Repositories

### DISCUSSION THREAD

**Discussion by Nazifa-Mosharrat** [community/community#140332] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a concern about contribution visibility in private repositories. When users lose access (e.g. after leaving a company), their contributions disappear from their profile, making it appear as though they didn’t contribute during that time. This impacts developers’ ability to showcase their work. While I understand the need for privacy, could GitHub implement a feature to preserve the visibility of a developer's contributions, even after access is revoked, without revealing any sensitive information? This would better reflect a developer’s work history while maintaining repository confidentiality.

**Comment by 20manas** [comment]
Yeah, this is what the behaviour should be. Otherwise, the contributions chart is unreliable and has no point since the contributions always disappear whenever a user changes their company.

**Comment by Jaygot5500-sudo** [comment]
N
